### ISSUE TITLE
Dedicated (Open)Metrics values configuration for XRay and Insight

### ISSUE THREAD

**Issue by mansur-abdirimov** [jfrog/charts#1604]
**Is this a request for help?**: No 

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): 
FEATURE REQUEST

**Which chart**: 
jfrog-platform, xray, insight

**Description**:
To take full advantage of Insight / Trends dashboard we need to enable (Open)Metrics feed for each of the products: Artifactory, XRay, Insight. Artifactory configuration allows to do this in a very convenient manner through dedicated set of values:
```
artifactory:
    artifactory:
        openMetrics:
            enabled: true
            filebeat:
                enabled: true
                elasticsearch:
                    url: http://jfrog-platform-insight:8082/elasticsearch
```
The same set of dedicated values are not present for XRay and Insight. It seems that the only way to enable this is through `systemYaml` values. Using `systemYaml` for this purpose creates an overhead as if you are to use it you are forced to define the rest of the system configuration. 

**Proposal**:
Enable (Open)Metrics configuration for XRay and Insight through dedicated set of values like it is done for Artifactory.

**Comment by chukka** [comment]
@mansur-abdirimov Thanks for this feature, we will evaluate this internally and update soon.

**Comment by chukka** [comment]
This feature  is available in  xray (>= 103.55.2)  via  #1664  and insight will be released in next minor version
